src/main.rs

```diff
@@ -66,10 +70,31 @@ fn write_err(err: StratisError) -> StratisResult<()> {
 }
 
 fn main() {
-
-    let engine = Rc::new(RefCell::new(SimEngine::new()));
-    // TODO: add cmdline option to specify engine
-    //  let context = Rc::new(RefCell::new(Context::new()));
+    let matches = App::new("stratis")
+        .version(&crate_version!())
+        .about("Stratis storage management")
+        .arg(Arg::with_name("debug")
+            .short("d")
```

Comment:
I don't anticipate having a lot of options for stratisd. The only people who will be running it directly are developers or testers. That's an argument that we won't have so many options so it'd be ok to have short options w/o collisions :smile: but also it's an argument against them, since we don't care if devs type a few more chars. I'll remove the short option.